posts/2023-10-18-announcing-the-new-rust-project-directors.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Announcing the New Rust Project Directors"
+author: Leadership Council
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+
+We are happy to announce that we have completed the process to elect new Project Directors.
+
+The new Project Directors are:
+
+* [Scott McMurray](https://github.com/scottmcm)
+* [Jakob Degen](https://github.com/JakobDegen)
+* [Santiago Pastorino](https://github.com/spastorino)
+
+They will join [Ryan Levick] and [Mark Rousskov] to make up the five members of the Rust Foundation Board of Directors who represent the Rust Project.
+
+The board is made up Project Directors, who come from and represent the Rust Project, and Member Directors who represent the corporate members of the Rust Foundation.
+These groups of Directors each, as a block, have equal voting power.

```

Comment:
```suggestion
Both of these director groups have equal voting power.
```

Is this true? I am not sure I understand what `as a block` here means. If this is right, I think it reads better.

Reply:
In the bylaws, a supermajority vote requires approval by 2/3 of Project Directors and 2/3 of Corporate Directors. By counting those separately, they are given equal weight, despite having fewer PDs.